main.c

```diff
@@ -16,6 +16,10 @@ how to use the page table and disk interfaces.
 #include <errno.h>
 #include <stdbool.h>
 
+int num_faults = 0;
+int num_reads = 0;
+int num_writes = 0;
+
```

Comment:
so it would be easier to understand this PR if this wasn't here, cause this isn't strictly related to this change right?

Reply:
I changed the base to `metrics` so I can test this with the code on that branch btw

Reply:
Yeah sorry about that. I was using the metrics to verify that it worked, but I should have removed them.

Reply:
Now since I changed this to be on `master`, I shouldn't see code changes that are already pushed to `metrics` here, but I do. Weird.